dist/aem.js

```diff
@@ -519,6 +519,26 @@ async function fetchPlaceholders(prefix = 'default') {
   return window.placeholders[`${prefix}`];
 }
 
+/**
+ * Wrap inline text content of block cells within a <p> tag.
+ * @param {Element} block the block element
+ */
+// eslint-disable-next-line import/prefer-default-export
+function wrapTextNodes(block) {
+  block.querySelectorAll(':scope > div > div').forEach((blockColumn) => {
+    if (blockColumn.hasChildNodes()) {
+      const hasWrapper = !!blockColumn.querySelector('picture')
+        || /* exclude certain elements from being wrapped */ (!!blockColumn.firstElementChild
+          && window.getComputedStyle(blockColumn.firstElementChild).display === 'block');
+      if (!hasWrapper) {
+        const p = document.createElement('p');
```

Comment:
There are case where we would have to wrap elements in a `<pre>` instead, e.g. if you have a single `code` child in the cell.